src/runtime/components/nuxt-icon.vue

```diff
@@ -13,16 +14,14 @@ const props = defineProps({
     required: false
   }
 })
-const icon = ref('')
+const icon:Ref<Record<string, any> | string> = ref('')
```

Comment:
I think the `icon` shouldn't be a `Record<string, any>`, only string. 

```
 ERROR(vue-tsc)  Type 'Record<string, any>' is not assignable to type 'string'.
```
The Record type here is probably leaked from the [import.meta.glob](https://vitejs.dev/guide/features.html#glob-import) below. It'd better to handle that before assigning to the `icon` value

Reply:
Please approve this line of code.